src/wikigraph.cc

```diff
@@ -35,24 +40,71 @@ WikiGraph::WikiGraph(const std::string& file_name) {
   }
 }
 
-
 // --------- Algorithms ------------
 // TODO: getPathBFS()
-std::vector<std::string> WikiGraph::getPathBFS(const std::string& from_page, const std::string& to_page) const {
-  std::vector<std::string> path_to_return;
-  // ...
-  (void) from_page;
-  (void) to_page;
+std::vector<std::string> WikiGraph::getPathBFS(
+    const std::string& start_page, const std::string& end_page) const {
+  std::cout << "Finding BFS Path from [" << start_page << "] to [" <<
```

Comment:
remove cout from prod; the user knows the start and end and algorithm bc they typed in the command, it doesn't need to be said back to them